apps/docs/src/.vitepress/components/IconLibraryItem.vue

```diff
@@ -0,0 +1,75 @@
+<script lang="ts" setup>
+import OnyxIcon from "~components/OnyxIcon/OnyxIcon.vue";
+import type { EnrichedIcon } from "../utils-icons";
+import { ref } from "vue";
+
+const props = defineProps<{
+  icon: EnrichedIcon;
+}>();
+
+const isCopied = ref(false);
+
+const handleCopy = async () => {
+  const { importName, iconName } = props.icon;
+  await navigator.clipboard.writeText(
+    `import ${importName} from "@sit-onyx/icons/${iconName}.svg?raw";`,
+  );
+  isCopied.value = true;
+  setTimeout(() => (isCopied.value = false), 3000);
+};
+</script>
+
+<template>
+  <div class="icon" tabindex="0" @click="handleCopy" @keyup.enter="handleCopy">
```

Comment:
If its clickable, I'd prefer to use a button instead. So we can also get rid of the tabindex and the keyup.enter

Reply:
nice! thanks! done :)